source/dub/commandline.d

```diff
@@ -898,7 +898,7 @@ class InitCommand : Command {
 			p.description = input("Description", p.description);
 			p.authors = input("Author name", author).split(",").map!(a => a.strip).array;
 			p.license = input("License", p.license);
-			string copyrightString = .format("Copyright © %s, %-(%s, %)", Clock.currTime().year, p.authors);
+			string copyrightString = format("Copyright © %s, %-(%s, %)", Clock.currTime().year, p.authors);
```

Comment:
Unrelated

Reply:
agree, it's unrelated, but in other place it was a question https://github.com/dlang/dub/pull/1980#issuecomment-665205310, I was think it can be fixed in this place too (I rewert it in next commit)